packages/pinia/src/store.ts

```diff
@@ -879,6 +879,12 @@ export function defineStore(
   } else {
     options = idOrOptions
     id = idOrOptions.id
+
+    if (__DEV__ && typeof id !== 'string') {
+      throw new Error(
+        `[🍍]: defineStore must be passed an id string, either as its first argument or via the "id" option.`
```

Comment:
I might deprecate the `id` option in the future since the `id` is always required anyway so I prefer to recommend only the argument way

```suggestion
        `[🍍]: "defineStore()" must be passed a store id as its first argument.`
```